odetoolbox/__init__.py

```diff
@@ -110,12 +95,13 @@ def _from_json_to_shapes(indict, options_dict, parameters=None) -> List[Shape]:
     all_parameter_symbols = set()
     all_variable_symbols_ = set()
     for shape_json in indict["dynamics"]:
-        shape = Shape.from_json(shape_json, time_symbol=options_dict["input_time_symbol"], differential_order_symbol=options_dict["differential_order_symbol"], parameters=parameters)
+        shape = Shape.from_json(shape_json, time_symbol=Config().input_time_symbol, parameters=parameters)
```

Comment:
The `time_symbol` argument isn't used in the `from_json` function. Can we remove it?